README.md

```diff
@@ -164,7 +164,7 @@ Validator                               | Description
 **isSlug**                              | Check if the string is of type slug. `Options` allow a single hyphen between string. e.g. [`cn-cn`, `cn-c-c`]
 **isStrongPassword(str [, options])**   | Check if a password is strong or not. Allows for custom requirements or scoring rules. If `returnScore` is true, then the function returns an integer score for the password rather than a boolean.<br/>Default options: <br/>`{ minLength: 8, minLowercase: 1, minUppercase: 1, minNumbers: 1, minSymbols: 1, returnScore: false, pointsPerUnique: 1, pointsPerRepeat: 0.5, pointsForContainingLower: 10, pointsForContainingUpper: 10, pointsForContainingNumber: 10, pointsForContainingSymbol: 10 }`
 **isTime(str [, options])**             | Check if the input is a valid time. e.g. [`23:01:59`, new Date().toLocaleTimeString()].<br/><br/> `options` is an object which can contain the keys `hourFormat` or `mode`.<br/><br/>`hourFormat` is a key and defaults to `'hour24'`.<br/><br/>`mode` is a key and defaults to `'default'`. <br/><br/>`hourFomat` can contain the values `'hour12'` or `'hour24'`, `'hour24'` will validate hours in 24 format and `'hour12'` will validate hours in 12 format. <br/><br/>`mode` can contain the values `'default'` or `'withSeconds'`, `'default'` will validate `HH:MM` format, `'withSeconds'` will validate the `HH:MM:SS` format.
-**isTaxID(str, locale)**                | Check if the given value is a valid Tax Identification Number. Default locale is `en-US`.<br/><br/>More info about exact TIN support can be found in `src/lib/isTaxID.js`<br/><br/>Supported locales: `[ 'bg-BG', 'cs-CZ', 'de-AT', 'de-DE', 'dk-DK', 'el-CY', 'el-GR', 'en-CA', 'en-GB', 'en-IE', 'en-US', 'es-ES', 'et-EE', 'fi-FI', 'fr-BE', 'fr-CA', 'fr-FR', 'fr-LU', 'hr-HR', 'hu-HU', 'it-IT', 'lb-LU', 'lt-LT', 'lv-LV' 'mt-MT', 'nl-BE', 'nl-NL', 'pl-PL', 'pt-BR', 'pt-PT', 'ro-RO', 'sk-SK', 'sl-SI', 'sv-SE' ]`
+**isTaxID(str, locale)**                | Check if the given value is a valid Tax Identification Number. Default locale is `en-US`.<br/><br/>More info about exact TIN support can be found in `src/lib/isTaxID.js`<br/><br/>Supported locales: `[ 'bg-BG', 'cs-CZ', 'de-AT', 'de-DE', 'dk-DK', 'el-CY', 'el-GR', 'en-CA', 'en-GB', 'en-IE', 'en-IN-GSTIN', 'en-US', 'es-ES', 'et-EE', 'fi-FI', 'fr-BE', 'fr-CA', 'fr-FR', 'fr-LU', 'hr-HR', 'hu-HU', 'it-IT', 'lb-LU', 'lt-LT', 'lv-LV' 'mt-MT', 'nl-BE', 'nl-NL', 'pl-PL', 'pt-BR', 'pt-PT', 'ro-RO', 'sk-SK', 'sl-SI', 'sv-SE' ]`
```

Comment:
I'm not so sure about the usage of `en-IN-GSTIN` (and `en-IN-PAN` in a future PR) since it does not follow the format of other locales. Maybe we should add a `options` object that has a property to specify which exact validator is used? And if that is not defined it would check both GSTIN and PAN?

I'm unsure on the naming of the option, so an option is also to have two boolean properties. One called `gstin` and one called `pan` which only change the behaviour of `isTaxID` when using the 'en-IN' locale.

Does that make sense? It would bring additional complexity to `isTaxID` but I think that adding this to the locale does not fall under consistent design.

Reply:
Hi @WikiRik 
You are right! How about this, we could create an object `multipleTinLocale` where we could add locale and their localeOptions. Something like this,
```
const multipleTinLocale = {
  'en-IN': {
    GSTIN: true,
    PAN: true,
  },
};
``` 
We will maintain the format of the locale in `taxIdFormat` object but for locales with multiple Tin we shall specify them as an object.
```
const taxIdFormat = {
  'en-IE': /^\d{7}[A-W][A-IW]{0,1}$/i,
  'en-IN': {
    GSTIN: /^((?!39)(?!00)[0-3][0-9]|97)([A-Z]{3}[ABCFGHLJPT][A-Z](?!0000)[0-9]{4}[A-Z])[1-9A-Z]Z[0-9 A-Z]$/,
    PAN: /^[A-Z]{3}[ABCFGHLJPT][A-Z](?!0000)[0-9]{4}[A-Z]$/,
  },
  'en-US': /^\d{2}[- ]{0,1}\d{7}$/,
}
```

Moving to the `isTaxId` Validator function, we shall add a parameter called `localOption` and we will be doing the following inside the function.

1. Check if the provided `locale` is in `multipleTinLocale`, if true then
2. Check if `localeOption` is provided, if false _throw Error locale requires localeOption_, if true then
3. Check if provided `localeOption` is in `multipleTinLocale` Object, if false _throw Error incorrect localeOption_, if true then,
4. Call the validator based on the provided `locale` and `localeOption`.

This way we will not be causing any impact to the existing functionality, and if done right we will be bringing the ability to validate locale's with multiple TIN.

Reply:
I think that sounds good. Would `localeOption` accept a string with the chosen validator or could it also accept an object to use both validators?

Reply:
Hi @WikiRik 
I planned of having it specific to string, because I am not sure on a use case where we would be calling both the validators on the same tax id .

Our function would look like
```
export default function isTaxID(str:String, locale: String,localeOption: String){

}

```

Reply:
Looking at other validators like `isIP` there are some cases where we check for multiple options but I do not know if GSTIN and PAN are that compatible or will just result in too many false positives. I think that a string for now is also fine

Reply:
Cool ! 😄 I am done with the code changes on the validator, working on the tests. Will submit for a review soon.

Thanks for the suggestions 😄👍